docker/local/polygon-edge.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+POLYGON_EDGE_BIN=./polygon-edge
+
+case "$1" in
+
+   "init")
+      echo "Generating secrets..."
+      node1id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data1 | grep Node | awk -F ' ' '{print $4}')
+      node2id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data2 | grep Node | awk -F ' ' '{print $4}')
+      node3id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data3 | grep Node | awk -F ' ' '{print $4}')
```

Comment:
not necessary to store the id here and in the next line, we're not going to use it